integrations/java/openhouse-java-runtime/build.gradle

```diff
@@ -41,6 +42,7 @@ dependencies {
     exclude group: 'com.zaxxer', module: 'HikariCP-java7'
     exclude group: 'org.apache.commons', module: 'commons-lang3'
   }
+  implementation("com.github.ben-manes.caffeine:caffeine:" + caffeineVersion)
 }
```

Comment:
can we also do output jar check to see if the newly added library is appropriately relocated? the output jar should look as follows: 
![image](https://github.com/user-attachments/assets/21a0b95c-bcab-497b-9103-afca08969240)